src/bigquery.ts

```diff
@@ -1826,7 +1934,9 @@ export class BigQuery extends common.Service {
       query.job.getQueryResults(query, callback as QueryRowsCallback);
       return;
     }
-    this.query(query, {autoPaginate: false}, callback);
+    const wrapIntegers = query.wrapIntegers ? query.wrapIntegers : false;
```

Comment:
I think we can add an options object to this function to match how `query()` expects `wrapIntegers`.